wezterm-input-types/src/lib.rs

```diff
@@ -1275,6 +1275,7 @@ bitflags! {
     pub struct WindowDecorations: u8 {
         const TITLE = 1;
         const RESIZE = 2;
+        const FANCY = 4;
```

Comment:
I wonder if we should rename `FANCY` to `INTEGRATED_BUTTONS` so that is a bit more descriptive of the function?

Reply:
We can name it also CSD (client-side decoration) it's shorten to write and that's how it regurarly named: https://en.wikipedia.org/wiki/Client-side_decoration

but wayland version already draws client side decoration